src/DGtal/io/doc/moduleDisplay3D.dox

```diff
@@ -199,9 +199,9 @@ And then visualize the resulting obj and mtl file by using for instance blender:
 
 @note You do not need a Board3D to export an OBJ file. There are
 custom methods to export OBJ files for some classes:
-    - \ref Mesh: use MeshWriter::export2OBJ
-    - \ref TriangulatedSurface: use MeshHelpers::exportOBJ, and MeshHelpers::exportOBJwithFaceNormalAndColor 
-    - \ref PolygonalSurface: use MeshHelpers::exportOBJ, and MeshHelpers::exportOBJwithFaceNormalAndColor 
+    - Mesh: use MeshWriter::export2OBJ
+    - TriangulatedSurface: use MeshHelpers::exportOBJ, and MeshHelpers::exportOBJwithFaceNormalAndColor 
+    - PolygonalSurface: use MeshHelpers::exportOBJ, and MeshHelpers::exportOBJwithFaceNormalAndColor 
```

Comment:
It is a little bit too late, but you can still use `ref` by specifying an empty string as second parameter:
```
    - \ref Mesh "": use MeshWriter::export2OBJ
```

Reply:
That's why it was not working. It seems \ref was perturbed by the ':' afterwards. Thanks for the trick !